app/assets/stylesheets/application.css.scss

```diff
@@ -119,3 +117,65 @@ a:hover {
 #footer {
     background:#000;
 }
+
+input[type="text"], input[type="password"], input[type="number"] {
```

Comment:
Avoid qualifying attribute selectors with an element.<br>Each selector in a comma sequence should be on its own line